dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowRegressionIT.java

```diff
@@ -54,8 +54,7 @@
 import static io.dockstore.common.CommonTestUtilities.version;
 
 /**
- * This test suite will have tests for the workflow mode of the Dockstore Client.
- * Created by aduncan on 05/04/16.
+ * This test suite will have tests for the workflow mode of the old Dockstore Client.
```

Comment:
Again, what is the old version? 
When was this added? 
Make use of the `@since` tag and `@author` tag to lend some structure to this.